fetch.bs

```diff
@@ -3237,7 +3250,102 @@ through TLS using ALPN. The protocol cannot be spoofed through HTTP requests in
 </ol>
 </div>
 
+<h3 id=https-upgrades>HTTPS upgrading</h3>
 
+<div algorithm="https-upgrades">
```

Comment:
This contains two algorithms, that doesn't work. I recommend following the style we use elsewhere for these.

Reply:
Changed to two separate algorithm divs